Makefile

```diff
@@ -37,7 +37,7 @@ prereq: make-requirements.txt
 #	pyenv global system $(PY34) $(PY35) $(PY36) $(PY37)
 	pyenv global system $(PY35) $(PY36) $(PY37)
 	@# Ensure all Python versions are registered for this project
-	@eval "$$(pyenv init -)" && pyenv versions --bare > .python-version;
+	@awk -F'=' '{print $$2 > ".python-version"}' make-requirements.txt
```

Comment:
This works, but IMHO it is a bit strange to do a file redirection _inside_ awk for this use case;
would it be simpler to
`@awk -F'=' '{print $$2}' make-requirements.txt >.python-version`

i.e. redirect the final result of awk to `.python-version`?

Reply:
I've change it now.